client/src/components/Markdown/Editor/CellWrapper.vue

```diff
@@ -1,12 +1,11 @@
 <template>
     <div @mouseenter="hover = true" @mouseleave="hover = false">
         <div class="d-flex">
-            <div
-                class="d-flex flex-column cursor-pointer"
-                :class="{ 'cell-wrapper-hover': hover }"
-                @click="$emit('toggle')">
-                <CellButton v-if="toggle" title="Collapse" :icon="faAngleDoubleUp" />
-                <CellButton v-else title="Expand" :icon="faAngleDoubleDown" />
+            <div class="d-flex cursor-pointer" :class="{ 'cell-wrapper-hover': hover }" @click="$emit('toggle')">
```

Comment:
Please avoid interaction handlers on static elements, as they are not accessible